### DISCUSSION TITLE
2005/05/30/%E6%88%91%E6%8D%A2%E4%BA%86%E6%96%B0%E6%89%8B%E6%9C%BA

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#32] [Announcements]
# 2005/05/30/%E6%88%91%E6%8D%A2%E4%BA%86%E6%96%B0%E6%89%8B%E6%9C%BA

原来的破阿尔卡特有一段时间没用，就干脆打不开了。

https://blog.qizhen.xyz/2005/05/30/%E6%88%91%E6%8D%A2%E4%BA%86%E6%96%B0%E6%89%8B%E6%9C%BA

**Comment by ruanqizhen** [comment]
珍珠变鱼眼
2005年06月14日 16:48 编辑
我喜欢这款手机——“新mini机王”

回复
倩
2005年06月2日 22:43 编辑
索爱太多人用了……还是喜欢松下……

回复
权育
2005年06月1日 21:07 编辑
索爱K700C不错啊,功能都比较齐,不过我的还是T610,呵呵.

回复
雲想
2005年05月31日 15:19 编辑
我以前有一部SIMENS的，一个字”烂”。还是觉得诺基亚的手机比较好

回复
米
2005年05月31日 10:48 编辑
手机我还是终爱SIMENS不过现在索爱好像太火爆了……

回复
哈米の哟哟
2005年05月30日 22:55 编辑
用几天你就会后悔了 挖哈哈哈哈 – -我食量大怎么着。。

回复
★ⓔ小ⓥⓐ㆝戒☀♨☯♠
2005年05月30日 20:02 编辑
? 哪啊 發佈者 ruanqizhen↑在這裡http://tw.match.yahoo.com/profile?id=personals-1035451489-134415

回复
